src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -169,148 +166,111 @@ Nethermind also has [detailed documentation](https://docs.nethermind.io), strong
 
-For further details, check out the tutorial on [running Geth light node](/developers/tutorials/run-light-node-geth/).
+Visit [Prysm docs](https://docs.prylabs.network/docs/getting-started/) to learn more. 
 
-**Setting up full sync with archive in [Besu](https://besu.hyperledger.org/)**
+### Synchronization modes {#sync-modes}
 
-`besu --sync-mode=FULL`
+To follow and verify current data in the network, the Ethereum client needs to sync with the latest network state. This is done by downloading data from peers, cryptographically verifying their integrity, and building a local blockchain database.
 
-Like any other configuration, it can be defined with the startup flag or in the config file. Another example is [Nethermind](https://docs.nethermind.io/nethermind/) which prompts you to choose a configuration during first initialization and creates a config file.
+Synchronization modes represent different approaches to this process with various trade-offs. Clients also vary in their implementation of sync algorithms. Always refer to the official documentation of your chosen client for specifics on implementation.
 
-## Consensus clients (formerly 'Eth2' clients) {#consensus-clients}
+#### Full sync {#full-sync}
 
-There are multiple consensus clients (previously known as 'Eth2' clients) to support the [consensus upgrades](/upgrades/beacon-chain/). They are running the Beacon Chain and will provide proof-of-stake consensus mechanism to execution clients after [The Merge](/upgrades/merge/).
+Full sync downloads all blocks (including headers, transactions, and receipts) and generates the state of the blockchain incrementally by executing every block from genesis.
 
-[View consensus clients](/upgrades/get-involved/#clients).
+- Minimizes trust and offers the highest security by verifying every transaction.
+- With an increasing number of transactions, it can take days to weeks to process all transactions.
 
-| Client                                                      | Language   | Operating systems     | Networks                              |
-| ----------------------------------------------------------- | ---------- | --------------------- | ------------------------------------- |
-| [Teku](https://pegasys.tech/teku)                           | Java       | Linux, Windows, macOS | Beacon Chain, Prater                  |
-| [Nimbus](https://nimbus.team/)                              | Nim        | Linux, Windows, macOS | Beacon Chain, Prater                  |
-| [Lighthouse](https://lighthouse-book.sigmaprime.io/)        | Rust       | Linux, Windows, macOS | Beacon Chain, Prater, Pyrmont         |
-| [Lodestar](https://lodestar.chainsafe.io/)                  | TypeScript | Linux, Windows, macOS | Beacon Chain, Prater                  |
-| [Prysm](https://docs.prylabs.network/docs/getting-started/) | Go         | Linux, Windows, macOS | Beacon Chain, Gnosis, Prater, Pyrmont |
+#### Fast sync
```

Comment:
I would mention that Snap Sync is more commonly used now, and perhaps switch their order in the page.